src/07-even-more-loops.js

```diff
@@ -109,7 +177,16 @@ function findSmallest(array) {}
  *  findLargest([ 0, 11, -2, 5 ]);
  *  //> 11
  */
-function findLargest(array) {}
+function findLargest(array) {
+  let largest = -Infinity;
```

Comment:
Once again lets set the value to our initial array item and not use ` Infinity `